onnxruntime/core/framework/tensor.cc

```diff
@@ -47,6 +47,21 @@ size_t Tensor::SizeInBytes() const {
   return ret;
 }
 
+Tensor Tensor::Clone(AllocatorPtr allocator) const {
+  Tensor cloned(dtype_, shape_, allocator, byte_offset_);
+  void* cloned_data_buffer = cloned.MutableDataRaw();
+
+  if (!IsDataTypeString()) {
+    std::memcpy(cloned_data_buffer, p_data_, SizeInBytes());
+  } else {
+    auto* ptr = static_cast<std::string*>(p_data_);
+    int64_t len = shape_.Size();
+    std::copy(ptr, ptr + len, static_cast<std::string*>(cloned_data_buffer));
+  }
```

Comment:
Does something test this? Doesn't look quite right given Tensor::Init does a placement 'new' for each string.

Reply:
I intended to check-in a test for this but forgot. Fixed.

Reply:
Cloning functionality is not needed and hence this got removed entirely.